image.yaml

```diff
@@ -110,17 +114,19 @@ modules:
           - name: os-java-jolokia
           - name: jolokia
           - name: os-eap7-openshift
-          - name: os-eap71-openshift
           - name: os-eap7-modules
-          - name: os-eap71-modules
+          - name: jboss.eap.cd.openshift.modules
           - name: os-eap7-ping
           - name: os-eap-activemq-rar
           - name: os-java-run
           - name: os-eap-launch
           - name: os-eap7-launch
+          - name: jboss.eap.cd.openshift.launch
+          - name: os-logging
```

Comment:
This module is repeated below.

Reply:
Ah, yeah, os-logging.

Reply:
@bstansberry fixed to match the eap71 order.